sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/StatsbeatConstants.cs

```diff
@@ -73,5 +73,9 @@ internal static class StatsbeatConstants
         /// 24 hrs == 86400000 milliseconds.
         /// </summary>
         internal const int AttachStatsbeatInterval = 86400000;
+
+        internal const string AttachStatsbeatMeterName = "AttachStatsbeatMeter";
```

Comment:
can you use the same meter for all statsbeat telemetry? so that you don't need to create NetworkStatsbeatMeter or FeatureStatsbeatmeter later?

```suggestion
        internal const string StatsbeatMeterName = "StatsbeatMeter";
```

Reply:
No, I will need a separate meterprovider for feature and network as they will be on different schedule. If I use the same meter, we will end up sending data multiple times